.github/workflows/test_branches.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           skip_doctest: 1
           TARGET: linux
           PYENV: conda
-          PACKAGES: mpi4py openmpi
+          PACKAGES: mpi4py
```

Comment:
I think if you are removing openmpi, you can / should also remote mpi4py.

Reply:
Correction: we need to leave openmpi and mpi4py here.  It can be removed from the `ubuntu-latest/3.9/conda` above.